packages/fxa-auth-server/lib/error.js

```diff
@@ -267,10 +267,7 @@ AppError.translate = function (request, response) {
       );
     }
   } else if (payload.validation) {
-    if (
-      payload.message &&
-      payload.message.includes('is required')
-    ) {
+    if (payload.message && payload.message.includes('is required')) {
```

Comment:
Nit: use `payload.message?.includes(…)`?

```suggestion
    if (payload.message?.includes('is required')) {
```

Reply:
I feel like we discussed this before.... it probably did not get updated during the joi fiasco. I'll update this, thanks!